modules/ui/raw_tag_editor.js

```diff
@@ -175,10 +175,34 @@ export function uiRawTagEditor(context) {
                     bindTypeahead(key, value);
                 }
 
+                var isRelation = (_entityID && context.entity(_entityID).type === 'relation');
+                var reference;
+
                 // Hoot: Override tag reference with tag copy
                 var select = uiTagSelectCopy(context);
 
                 row.call( select );
```

Comment:
Should call this after other button types or move checkmark above column

![image](https://user-images.githubusercontent.com/2600932/82495618-5e4db680-9ab9-11ea-93e3-13f4db9d108d.png)